### DISCUSSION TITLE
Is there a way to add an additional font?

### DISCUSSION THREAD

**Discussion by radhaz** [parnic/ice-hud#38] [Q&A] [answered]
I started using the ShestakUI and I am trying to "unify" my UI for cohesiveness and ShestakUI uses the Pixel font, is there a way I can have IceHUD use that font as well?

**Comment by parnic** [comment]
Yes, it should show up under /icehud -> Font Settings. I use LibSharedMedia 3.0, which is supposed to be a central repository of all available fonts.

<img width="686" alt="image" src="https://github.com/parnic/ice-hud/assets/888126/b1ecf0fa-5efb-4d3f-86f9-80f29d78b3ff">

**Reply by radhaz** [reply]
I think there's a miscommunication on my part as I should have specified I found that panel/setting in the config. The font used by ShestakUI is not in the list, is there a way that I as an end user can add it to that list?

**Reply by parnic** [reply, marked as answer]
You might try installing this mod? https://www.curseforge.com/wow/addons/sharedmedia

I'm not sure - I'm using a library that's supposed to collect all the fonts and provide them to any addons that want to use them. Maybe ShestakUI should be adding its font to the shared list? I don't know what else I could do here to support extensible fonts.

**Reply by radhaz** [reply]
You don't need to do anything really, I was just asking for guidance how to change the iceHUD font to a different font and you gave it (by using the sharedmedia mod).  Thank you very much for maintaining the mod and your help here.
